src/lib/i18n.ts

```diff
@@ -1,47 +1,40 @@
 import i18n from "i18next";
+import LanguageDetector from "i18next-browser-languagedetector";
 import Backend from "i18next-http-backend";
 import {
-	type UseTranslationResponse,
 	initReactI18next,
 	useTranslation,
 } from "react-i18next";
 
-const supportedLanguages = ["nl-NL", "en-US"] as const;
+// Authomatically detect supported languages
+const supportedLocales = Object.keys(
+	import.meta.glob("/public/i18n/*.json", {
+		query: "?url",
+		import: "default",
+	}),
+).map((path) =>
+	path.replaceAll(/\/public\/i18n\/|\.json/g, ""),
+);
 
-export const init = async () => {
-	await i18n
-		.use(
-			new Backend(null, {
-				loadPath: (languages) => `/i18n/${languages}.json`,
-			}),
-		)
+export const init = () =>
+	i18n
+		.use(Backend)
+		.use(LanguageDetector)
 		.use(initReactI18next)
 		.init({
-			supportedLngs: supportedLanguages,
-			fallbackLng: supportedLanguages,
-			lng: "en",
+			supportedLngs: supportedLocales,
+			fallbackLng: import.meta.env.PROD ? "en-US" : "dev",
```

Comment:
```suggestion
			fallbackLng: import.meta.env.PROD ? "nl-NL" : "dev",
```
cos we mostly have Dutch clients, right?